packages/shared/src/db/client.ts

```diff
@@ -0,0 +1,28 @@
+import type { QueryResult } from "@op-engineering/op-sqlite";
+import { BaseSQLiteDatabase } from "drizzle-orm/sqlite-core";
+import { SqliteRemoteResult } from "drizzle-orm/sqlite-proxy";
+import { Schema } from "./types";
+
+export type AnySqliteDatabase = BaseSQLiteDatabase<
```

Comment:
Do we know if this works with our anticipated web sqlite implementation?

Reply:
Yes it will, this type matches the base implementation of all the SQLite drivers. I tried it as just a union of the three, but there were type errors some where in Drizzle's internals. Will add a comment.

Reply:
Nice!